### DISCUSSION TITLE
Visual Studio integration does not work correctly with projects on UNC paths

### DISCUSSION THREAD

**Discussion by ThadHouse** [microsoft/vcpkg#19291] [Ideas]
I run vcpkg at work, where we use folder redirection from a server for the documents folder. VCPKG's integration does not seem to handle this, and gives the following error.

error MSB3021: Unable to copy file "Microsoft.PowerShell.Core\FileSystem::\\SERVER\Folder Redirection\USER\Documents\Concurrency Camp\CppAsync\Debug\libuv.dll" to "\\SERVER\Folder Redirection\USER\Documents\Concurrency Camp\CppAsync\Debug\libuv.dll". The given path's format is not supported.

**Comment by PhoebeHui** [comment]
@ThadHouse, could you help confirm if it still be a problem with vcpkg latest source?

**Comment by ThadHouse** [comment]
I'll try it out later this week.

**Comment by PhoebeHui** [comment]
@ThadHouse, do you have a chance to double confirm it?

**Comment by ThadHouse** [comment]
I'm so sorry I forgot to test. I have easier access to shares so I'll give it a quick try tomorrow.

**Comment by ThadHouse** [comment]
I just double checked with the latest build, and yes this is still a problem. I get the same error. To reproduce it you can just open a project directly from a file share. It doesn't explicitly have to be folder redirection, which makes the repro much easier.

**Comment by PhoebeHui** [comment]
@ThadHouse, thanks for confirm this problem!
